docs/product-user-guide/src/main/asciidoc/packages-create-proc.adoc

```diff
@@ -2,15 +2,15 @@
 
 = Creating a Package
 
-Rules and other assets must be collected into a package before they can be deployed. When a package is built, the assets contained in the package are validated and compiled into a deployable package. You can create individual packages of assets within Business Central.
+Rules and other assets must be collected into a package before they can be deployed. When a package is built, the assets contained in the package are validated and compiled into a deployable package. You can create individual packages of assets within {CENTRAL}.
```

Comment:
This isn't strictly correct. 

A Project is built and becomes a deployable "unit" (KJAR). The Project can contain multiple Packages.

You cannot create a deployable "unit" (KJAR) from a single Package.

Reply:
@michelehaglund or @mramendi, please note this and make the update accordingly when the time comes for you to use/update this content in your deliverables.